src/Tribe/Events/Attendees_List.php

```diff
@@ -22,44 +19,69 @@
 class Attendees_List {
 
 	/**
+	 * Meta key to hold the if the Post has Attendees List hidden.
+	 *
 	 * @since TBD
-	 * @var string Meta name to control whether the Attendee List
-	 *             meta was changed by a shortcode in the content.
+	 *
+	 * @var string
 	 */
-	protected static $attendee_list_by_shortcode = 'tribe_tickets_attendee_list_triggered_by_shortcode';
+	const HIDE_META_KEY = '_tribe_hide_attendees_list';
 
 	/**
-	 * Check if given event is hiding the attendees list.
+	 * Meta name to control whether the Attendee List meta was changed by a shortcode in the content.
 	 *
 	 * @since TBD
 	 *
-	 * @param int|WP_Post $post The Post being checked.
+	 * @var string
+	 */
+	protected static $attendee_list_by_shortcode = 'tribe_tickets_attendee_list_triggered_by_shortcode';
+
+	/**
+	 * Determine if we need to hide the attendees list.
+	 *
+	 * @since TBD
 	 *
-	 * @return bool True if event is hiding the attendees list. False otherwise.
+	 * @param int|WP_Post $post   The post object or ID.
+	 * @param boolean     $strict Whether to strictly check the meta value.
 	 *
-	 * @throws InvalidArgumentException Could not determine if given event is hiding or showing the Attendee List.
+	 * @return bool|null Whether the attendees list is hidden, null if not set at all with strict mode off.
 	 */
-	public function is_event_hiding_attendee_list( $post ) {
-		// Attendees list is a Plus feature, if ET Plus is not present it will always be hidden.
-		if ( ! class_exists( 'Tribe__Tickets_Plus__Main' ) ) {
-			return true;
-		}
-
+	public static function is_hidden_on( $post, $strict = true ) {
 		if ( is_numeric( $post ) ) {
 			$post = get_post( $post );
 		}
 
 		if ( ! $post instanceof WP_Post ) {
-			/*
-			 * There's no way to apply a sensible default in this scenario.
-			 *
-			 * Let's throw an exception so the client calling this function deals
-			 * with it according to the >> caller << context.
+			return true;
+		}
+
+		$is_hidden = get_post_meta( $post->ID, self::HIDE_META_KEY, true );
+
+		// By default non-existent meta will be an empty string
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: b92717283bfb0d7dec34650c93f3de22</sub>